PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m

```diff
@@ -20,41 +20,41 @@
 //  THE SOFTWARE.
 //
 
-#import "HyBidPrebidUtils.h"
+#import "HyBidHeaderBiddingUtils.h"
 
 NSString *const PNLiteKeyPN_BID = @"pn_bid";
 double const kECPMPointsDivider = 1000.0;
 
-@implementation HyBidPrebidUtils
```

Comment:
I kept the old one too since the partners that use in-app bidding also use this class. Let's hope it doesn't become an issue.

Reply:
Readded with deprecation warnings. Please check my last commit @eros902002 👍